Packs/XSOARStorage/Integrations/XSOARStorage/XSOARStorage.yml

```diff
@@ -0,0 +1,52 @@
+category: Utilities
+commonfields:
+  id: XSOAR Storage
+  version: -1
+configuration:
+- defaultvalue: "1024"
+  display: Max Size of Store in bytes (Maximum of 1024000)
+  name: maxsize
+  required: true
+  type: 0
+description: Facilitates the storage and retrieval of key/value pairs within XSOAR.
+display: XSOAR Storage
+name: XSOAR Storage
+script:
+  commands:
+  - arguments:
+    - description: The namespace to retrieve keys from.
+      name: namespace
+    description: List the keys available.
+    name: xsoar-store-list
+    outputs:
+    - contextPath: XSOAR.Store
```

Comment:
Looks like this command's real output path is XSOAR.Store.keys.
Also, please provide a type here